_posts/2017/2017-02-20-creating-a-sales-analytics-app-with-ruby-on-rails.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "Creating a Sales Analytics App with Ruby on Rails"
+date: "2017-02-20 07:00:00"
+image: blog-header/hamburg-port-rails.jpg
```

Comment:
I like the header image very much but on the blog detail page you don't see the bridge anymore.
Maybe one of those fit better:
https://www.pexels.com/photo/rails-train-path-straight-1425/
https://www.pexels.com/photo/sky-forest-rails-straight-981/
https://www.pexels.com/photo/mountains-nature-sunny-bridge-1018/